stripe/src/main/java/com/stripe/android/view/PaymentUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.stripe.android.R;
+
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+import java.util.Currency;
+import java.util.Locale;
+
+public class PaymentUtils {
+
+    /**
+     * Formats a monetary amount into a human friendly string where zero(-ish) is returned
+     * as free.
+     */
+    static String formatPriceStringUsingFree(@NonNull Context context, long amount, @NonNull Currency currency) {
+        if (amount == 0) {
+            return context.getResources().getString(R.string.price_free);
```

Comment:
This works -- I was wondering if you wanted to allow a `@StringRes int` in so if the amount is 0 (and the nice part about longs is that == is a-ok), you can return whatever that string res is. That way we can return "Free Shipping", "No Tax Applied", or whatever weird things we can think of